syncer/client.go

```diff
@@ -69,7 +69,28 @@ func (m *syncPeerClient) Close() {
 	}
 
 	close(m.peerStatusUpdateCh)
-	close(m.peerDisconnectCh)
+	close(m.peerConnectionUpdateCh)
+}
+
+func (m *syncPeerClient) GetPeerStatus(peerID peer.ID) (*NoForkPeer, error) {
+	clt, err := m.newSyncPeerClient(peerID)
+	if err != nil {
+		return nil, err
+	}
+
+	timeoutCtx, cancel := context.WithTimeout(context.Background(), defaultTimeoutForStatus)
+	defer cancel()
+
+	status, err := clt.GetStatus(timeoutCtx, &emptypb.Empty{})
+	if err != nil {
+		return nil, err
+	}
+
+	return &NoForkPeer{
+		ID:       peerID.String(),
```

Comment:
Not sure why, but this syntax was not working, looks like peerId.String return string which is used only for output I had to change to 
ID:       string(peerID),
to make it work, will investigate this further, just wanted to give you info

Reply:
Fixed 5b4810311e5b20ac5e2437b0ec48b85bdb4c8402